src/main/java/racingcar/Cars.java

```diff
@@ -0,0 +1,31 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class Cars {
+    private List<Car> cars;
+
+    public Cars(int carCount) {
+        makeCars(carCount);
+    }
+
+    public Stream<Car> stream(){
+        return cars.stream();
+    }
+
+    public void makeCars(int carCount) {
+        cars = new ArrayList<>();
+        for(int i = 0; i< carCount; i++){
+            cars.add(new Car());
+        }
```

Comment:
Java 8을 사용해 진행하고 있으니 Stream의 generate를 활용해보실수 있을 것 같네요~

```suggestion
         cars = Stream.generate(() -> new Car())
                    .limit(carCount)
                    .collect(Collectors.toList());
```